ReleaseTooling/Sources/ZipBuilder/main.swift

```diff
@@ -300,6 +300,14 @@ struct ZipBuilderTool: ParsableCommand {
       }
     } else {
       // Do a Firebase Zip Release package build.
+
+      // For the Firebase zip distribution, we disable version checking at install time by
+      // setting a high version to install. The minimum versions are controlled by each individual
+      // pod's podspec options.
+      PlatformMinimum.initialize(ios: "14.0",
+                                 macos: "11.0",
+                                 tvos: "14.0")
```

Comment:
Do we want to make a new static method on `PlatformMinimum` like `PlatformMinimum.useLatestMajorVersions()` or something and have those versions defined there instead of here?